### ISSUE TITLE
Perform core update before installing other dependencies.

### ISSUE THREAD

**Issue by ParitoshikPaul** [dbjpanda/drupal-on-docker#7]
composer outdated drupal/*

**Comment by dbjpanda** [comment]
Yes it should be updated automatically here in Github. Probably a mirror repo of official composer drupal project can help.  https://github.com/dbjpanda/drupal-on-docker/tree/master/docker/nginx
